### PR TITLE
feat: v0.9.0 blog post && docs changes

### PR DESCRIPTION
**Pull request by gmpinder**

### CODE REVIEW

**Comment by gmpinder** [comment]
Still have some work to fluff up the blog post

**Comment by fiftydinar** [comment]
I forgot to add `try` in `get_json_array` examples, so please edit that

File: src/content/docs/blog/v090-features-changes.mdx [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      github: xynydev
+---
+
+:::caution[TLDR]
+The BlueBuild CLI `v0.9.0` update introduces **breaking changes**. [Show me what you broke!](#breaking-changes)
+:::
+
+The BlueBuild CLI is the tool that builds all BlueBuild custom images. It is also installed on every image built with BlueBuild and can be used to build and tinker with images locally. Lately we _(gmpinder)_ have been hard at work on a new CLI version to automate many administrative tasks you have to do as a custom image creator. But enough of the fluff, let's get right to it!
+
+## New Features
```
Comment (gmpinder) [inline src/content/docs/blog/v090-features-changes.mdx:20]: We'll also want to add the new `rechunk` feature under here

**Comment by xynydev** [comment]
https://github.com/blue-build/template/pull/45#issuecomment-2522943018

**Review by xynydev — APPROVED** [review]

**Pull request merged**